src/cluster/src/delete.rs

```diff
@@ -206,28 +207,33 @@ impl ClusterUninstaller {
         let ns = &self.config.namespace;
 
         // delete objects
-        self.remove_objects("spugroups", ns, None)?;
-        self.remove_objects("spus", ns, None)?;
-        self.remove_objects("topics", ns, None)?;
-        self.remove_objects("persistentvolumeclaims", ns, Some("app=spu"))?;
+        self.remove_custom_objects("spugroups", ns, None)?;
+        self.remove_custom_objects("spus", ns, None)?;
+        self.remove_custom_objects("topics", ns, None)?;
+        self.remove_custom_objects("persistentvolumeclaims", ns, Some("app=spu"))?;
 
         // delete secrets
         self.remove_secrets("fluvio-ca")?;
         self.remove_secrets("fluvio-tls")?;
 
         self.remove_partitions(ns).await?;
-        self.remove_objects("partitions", ns, None)?;
+        self.remove_custom_objects("partitions", ns, None)?;
 
         Ok(())
     }
 
     /// Remove objects of specified type, namespace
-    fn remove_objects(
+    fn remove_custom_objects(
         &self,
         object_type: &str,
         namespace: &str,
         selector: Option<&str>,
     ) -> Result<(), UninstallError> {
+        // If fluvio-sys is not installed, there are no custom objects to remove
+        if !self.is_sys_installed()? {
```

Comment:
let's make it more less dependent on `sys-charts`.  Just try to remove it and ignore errors.

Reply:
sys installed check is repeated for every call, either only done once or just don't check

Reply:
I forgot to remove this when I made deletes non-error. Should be good now